Server.js

```diff
@@ -6,18 +6,21 @@ var winston = require('winston');
 var fs = require('fs');
 var tcpPortUsed = require('tcp-port-used');
 var http = require('http');
-var https = require('https');
 
-try {
-  fs.statSync('./config/config.js');
-} catch (error) {
-  console.log("Config file is not setup. Read README.md for config setup.");
-  return;
-}
-var config = require('./config/config');
-var models = require('./models');
-var log = require('./logging');
+var configPath = './config'
+process.argv.forEach((val, index) => {
+  if (val.toLowerCase().startsWith('config=')) {
+    configPath  = val.split('=')[1];
+  }
+});
+
+var allConfig = require('./cacconfig');
+allConfig.loadConfig(configPath);
+var config = allConfig.config;
```

Comment:
`loadConfig` should should attach the top level configuration fields it loads from the configuration file as attributes of the `cacconfig` module. This would avoid the need to do `allConfig.config` everywhere. 

This will just be a straightforward for loop of the config object created from parsing the configuration file.